### PR TITLE
Bugfix/logout fix

### PR DESCRIPTION
**Pull request by n13**
Logout was broken because login info was stored in a variable called currentAuthStatus

Fix: update currentAuthStatus on logout

### CODE REVIEW

File: lib/app.dart [hunk 1]
```diff
@@ -15,8 +15,6 @@ import 'package:hypha_wallet/ui/blocs/authentication/authentication_bloc.dart';
 import 'package:hypha_wallet/ui/blocs/deeplink/deeplink_bloc.dart';
```
Comment (n13) [inline lib/app.dart:15]: lint only

File: lib/core/network/repository/auth_repository.dart [hunk 2]
```diff
@@ -43,6 +43,8 @@ class AuthRepository {
     status.listen((AuthenticationStatus event) {
```
Comment (n13) [inline lib/core/network/repository/auth_repository.dart:43]: the fix

File: lib/core/network/repository/auth_repository.dart [hunk 3]
```diff
@@ -43,6 +43,8 @@ class AuthRepository {
...
       if (event is Authenticated) {
         currentAuthStatus = event;
+      } else if (event is UnAuthenticated) {
```
Comment (gguijarro-c-chwy) [inline lib/core/network/repository/auth_repository.dart:46]: we should remove the if completely, Just pass the event to the currentAuthStatus

Both if and else do the same

Comment (n13) [reply lib/core/network/repository/auth_repository.dart:46]: well there's also an unknown status. I guess nobody emits that so sure

**Review by gguijarro-c-chwy — APPROVED** [review]

File: lib/core/network/repository/auth_repository.dart (hunk 3 above)
Comment (n13) [reply lib/core/network/repository/auth_repository.dart:46]: Actually didn't feel like testing whether unauthorized can come through, but the if could still be better yea

Comment (n13) [reply lib/core/network/repository/auth_repository.dart:46]: I was just assuming there to be a reason for the if statement...

**Pull request merged**